be/src/exec/vectorized/sorting/sort_permute.h

```diff
@@ -24,6 +24,7 @@ struct PermutationItem {
     uint32_t permutation_index;
```

Comment:
Could we remove the `permutation_index` field currently?

Reply:
done